### DISCUSSION TITLE
Why TextSize != Bounds.Height?

### DISCUSSION THREAD

**Discussion by Aleksey1555** [mono/SkiaSharp#1765] [Q&A]
Hello!
my code is:
```
string str="Good Morning 1";
SKPaint textPaint = new SKPaint
{
      Color = SKColors.Black,
      TextSize=textsize // parameter, for example 10
 };
SKRect textBounds = new SKRect();
textPaint.MeasureText(str, ref textBounds);
// And here , i expected to textBounds.Height==textSize But it does not
// textBounds={Left=-1,Top=-9,Width=72.14746,Height=13};
```
So, textSize=10 and Height=13 !!
The documentation say's that TextSize - is a height ot text!. But this example show's another results. (((

i need revers method or algorithm which take specified width, and will return TextSize. 
Help

**Comment by charlenni** [comment]
I try to answer this: your text includes a "g", which goes below the normal ground line of text. This is specifically to the chosen font. So remove the "g" and try it again. I assume, that the text height is than 10.

**Reply by mattleibow** [reply]
I believe this is called the descent. This also applies to the width. The character may extend past its specified width in order to make text appear more pleasant.

**Comment by artemiusgreat** [comment]
More.  
https://stackoverflow.com/a/27631737/437393
